templates/wizard.html.twig

```diff
@@ -0,0 +1,63 @@
+{% extends '@Tabler/fullpage.html.twig' %}
+
+{% from '@Tabler/components/progress_bar.html.twig' import progress_bar %}
+{% from '@Tabler/components/button.html.twig' import button %}
+{% from '@Tabler/components/buttons.html.twig' import submit_button %}
+
+{% block page_content %}
+
+    <div
+            class="container {% block container_size %}container-narrow{% endblock %} {% block container_class %}py-4{% endblock %}"
+            {% block container_attr %}{% endblock %}
+    >
+        {% block wizard_before %}{% endblock %}
+
+        {% block wizard_logo %}
+            <div class="text-center mb-4">
+                <h1 class="{% block navbar_brand_classes %}navbar-brand {% block navbar_brand_class %}navbar-brand-autodark{% endblock %}{% endblock %}">
+                    {% include '@Tabler/includes/logo.html.twig' %}
+                </h1>
+            </div>
+        {% endblock %}
+
+        {% block wizard_header_before %}{% endblock %}
+        {% block wizard_header %}{% endblock %}
+        {% block wizard_header_after %}{% endblock %}
+
+        <div
+                class="card {% block card_class %}card-md{% endblock %}"
+                {% block card_attr %}{% endblock %}
+        >
+            {% block wizard_content_before %}{% endblock %}
+            {% block wizard_content %}{% endblock %}
+            {% block wizard_content_after %}{% endblock %}
+        </div>
+
+        {% block wizard_footer_before %}{% endblock %}
+        {% block wizard_footer %}
+            <div class="row align-items-center mt-3">
+                <div class="col-4">
+                    {% block progress_bar %}
+                        {{ progress_bar({current : 1, max : 3}) }}
+                    {% endblock %}
+                </div>
+                <div class="col">
+                    <div class="btn-list justify-content-end">
+                        {% block back_button %}
+                            {{ button(false, { title : 'Close'|trans({}, 'TablerBundle'), combined: true, url : '#' }, 'link') }}
+                        {% endblock %}
+
+                        {% block next_button %}
+                            {{ submit_button(false, { title : 'Show homepage'|trans({}, 'TablerBundle'), combined: true }, 'primary') }}
+                        {% endblock %}
+                    </div>
+                </div>
+            </div>
+        {% endblock %}
```

Comment:
1.Do we care about default block name ? Prefix with `wizard` ?

Reply:
Ah, after reading it 5 times, I get it... trans != trad
Yeah, why not adding new translations. 
I will add Weblate support, so the community can translate them.

First I was thinking if we should add some variables, eg. `progress_current` or `back_url` but I guess we have to override the blocks in the project anyway, so that would not cover much more use-cases ... but I have no immediate use-case and find it hard to imagine. 
But yes, my gut says add `wizard_` for these blocks.

Reply:
OK, I'll rename `back` to `previous` and add trans + prefix block names.
I can only do that next Monday though.

Reply:
Good to go 👍